components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -55,7 +55,7 @@ export function parseProps(workspaceId: string, search?: string): StartWorkspace
     const runsInIFrame = window.top !== window.self;
     return {
         workspaceId,
-        runsInIFrame: window.top !== window.self,
+        runsInIFrame,
```

Comment:
👋 hey @filiptronicek, why is this change necessary?

Reply:
No necessary, just reuses a variable we already had for simplicity. Happy to revert as it's unrelated to the change itself and I just happened to make it while investigating workspace starts.

Reply:
Oh, I see. 

It seems safe to keep scope wise, no need to revert (I should have looked above earlier).